src/pages/Overview/OverviewStatuses.tsx

```diff
@@ -33,9 +33,7 @@ class OverviewStatuses extends React.Component<Props> {
     }
     return (
       <>
-        <Link to={`/${targetPage}?namespaces=${name}`} >
-          {text}
-        </Link>
+        <Link to={`/${targetPage}?namespaces=${name}`}>{text}</Link>
```

Comment:
This is probably not relevant to this PR.

Reply:
It went after rebasing. This was changed in another PR.